data_structures/linked_list/test_linked_list.py

```diff
@@ -111,4 +113,23 @@ def test_insertAfter_empty(empty_new_list):
 def test_insertAfter_multiple(new_ll_iter):
     a = new_ll_iter
     a.insertAfter(2, 9)
-    assert a.find(9) is True      
\ No newline at end of file
+    assert a.find(9) is True      
```

Comment:
For these insert and append tests, you should also test that the values are being inserted in the correct position.